administration/src/cards/extensions/KoblenzReferenceNumberExtension.tsx

```diff
@@ -14,6 +14,7 @@ const KoblenzReferenceNumberMaxLength = 15
 class KoblenzReferenceNumberExtension extends Extension<KoblenzReferenceNumberState, null> {
   public readonly name = KoblenzReferenceNumberExtension.name
 
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Maybe this is not a great rule either :D But let's ignore it for now^^

Reply:
Will be fixed in #1667.